stat/init.lua

```diff
@@ -138,9 +138,11 @@ local function stat(args)
     do
         local i = box.stat.net()
         for k, v in pairs(i) do
-            stats['stat.net.' .. k:lower() .. '.total'] = v.total
-            stats['stat.net.' .. k:lower() .. '.rps'] = v.rps
-        end
+            if type(v) == "table" then
```

Comment:
Nit: single quotes is used around, so it would be looks better if you'll use single quotes here too.